code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -7,6 +6,10 @@
 	w_class = WEIGHT_CLASS_SMALL
 	icon = 'icons/mob/human_parts.dmi'
 	icon_state = ""
+	/// The icon for Organic limbs using greyscale
+	var/icon_greyscale = DEFAULT_BODYPART_ICON_ORGANIC
```

Comment:
while we're here, what?

Reply:
I set it to a dmi file here but fikou suggested I use the define

Reply:
oh good god they're just the same icon file